src/core/benches/gather.rs

```diff
@@ -0,0 +1,68 @@
+#[macro_use]
+extern crate criterion;
+
+use std::fs::File;
+use std::io::BufReader;
+use std::path::PathBuf;
+
+use sourmash::index::calculate_gather_stats;
+use sourmash::signature::Signature;
+use sourmash::sketch::Sketch;
+
+use criterion::{black_box, Criterion};
```

Comment:
... and add these here:
```suggestion
use criterion::{black_box, criterion_group, criterion_main, Criterion};
```